src/main/java/com/_up/megastore/services/implementations/CategoryService.java

```diff
@@ -37,4 +37,10 @@ private Category getSuperCategory(UUID superCategoryId) {
         return superCategoryId != null ? findCategoryByIdOrThrowException(superCategoryId) : null;
     }
 
+    @Override
+    public CategoryResponse restoreCategory(UUID categoryId){
+        Category category = findCategoryByIdOrThrowException(categoryId);
+        category.setDeleted(false);
+        return CategoryMapper.toCategoryResponse(categoryRepository.save(category));
+    }
```

Comment:
Para dejar todos los restore iguales podríamos agregar la lógica para evitar restaurar productos que no están borrados. Esto se hizo en #27 
```suggestion
    public CategoryResponse restoreCategory(UUID categoryId){
        validateCategoryForRestoration(categoryId);
        Category category = findCategoryByIdOrThrowException(categoryId);
        category.setDeleted(false);
        return CategoryMapper.toCategoryResponse(categoryRepository.save(category));
    }
```